### ISSUE TITLE
Doesn't work on iPhone

### ISSUE THREAD

**Issue by thechief389** [mat3e/brains#2]
Dragging the images on my iPhone makes the page scroll.

**Comment by mat3e** [comment]
For all the browsers? Which browser have you checked?

**Comment by thechief389** [comment]
Safari

**Comment by mat3e** [comment]
Unfortunately, there is a problem with this particular browser. It is pointed out e.g. as an issue in the library I use for drag&drop: https://github.com/bevacqua/dragula/issues/426. And I cannot disable all the scrolling, which could be a workaround :/

**Comment by MoralCode** [comment]
what about adding a dropdown menu somewhere to allow the order of the tiles to be set that way?

**Comment by thechief389** [comment]
> what about adding a dropdown menu somewhere to allow the order of the tiles to be set that way?

That could work.

**Comment by mat3e** [comment]
@MoralCode like dropdown e.g. above or below an image with numbers representing the indexes?

**Comment by MoralCode** [comment]
yeah. i think a full-width version of this part of the desktop site might look good on mobile 
![Screenshot_20200411_175407](https://user-images.githubusercontent.com/17362949/79058052-9988da00-7c1d-11ea-8eae-39000e082577.png)
The dropdown could be either the leftmost or rightmost item in each row and maybe it would be nice to also add some javascript to auto-rearrange the images to stay in the correct order when a user changes one of the dropdowns
